### ISSUE TITLE
chainedforeignkeyfeild don't render any values

### ISSUE THREAD

**Issue by ofathy** [jazzband/django-smart-selects#262]
You **MUST** use this template when reporting issues. Please make sure you follow the checklist and fill in all of the information sections below.

----------------

All versions of django-smart-selects prior to version 1.2.8 are vulnerable to an XSS attack as detailed in [issue 171](https://github.com/digi604/django-smart-selects/issues/171#issuecomment-276774103). As a result, all previous versions have been removed from PyPI to prevent users from installing insecure versions. All users are urged to upgrade as soon as possible.

## Checklist

<sup><sup>Put an `x` in the bracket when you have completed each task, like this: `[x]`</sup></sup>

- [x ] This issue is not about installing previous versions of django-smart-selects older than 1.2.8. I understand that previous versions are insecure and will not receive any support whatsoever.
- [ ] I have verified that that issue exists against the `master` branch of django-smart-selects.
- [ x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x] I have debugged the issue to the `smart_selects` app.
- [x ] I have reduced the issue to the simplest possible case.
- [ ] I have included all relevant sections of `models.py`, `forms.py`, and `views.py` with problems.
- [ x] I have used [GitHub Flavored Markdown](https://help.github.com/articles/creating-and-highlighting-code-blocks/) to style all of my posted code.

## Steps to reproduce

1. 
2. 
3.

**Comment by basilrabi** [comment]
In the branch `js-unlinting-fixes` in admin page, the 1st chained foreign key which is dependent on a foreign key is rendered correctly for me.
However,  the 2nd-degree (or more) chained foreign key field, which is dependent on the 1st chained foreign key field, is not rendered correctly. If the 2nd-degree (or more) chained foreign key field was previously edited or has a previously set `data-value` before loading the admin page, what is being rendered is the empty option and not the previously-set `value`.

#260 fixes this issue for me.

**Comment by shaistha25** [comment]
I'm using ChainedForeignKey for admin page. The values in the 2nd dropdown is not rendered at all. 
What is the fix?

**Comment by basilrabi** [comment]
Are you using `js-unlinting-fixes` branch?

**Comment by shaistha25** [comment]
I did pip install git+https://github.com/digi604/django-smart-selects.git@js-unlinting-fixes and even an upgrade from issue#253 solution.
I also did 'collect static' and added the chainedfk.js(), chainedm2m.js() to the Base.html of admin template.
Finally works like charm with IE and Firefox. But with Chrome it's the never ending error 'Uncaught ReferenceError: chainedfk is not defined' at line12 of bindfields.js. Nevertheless, javascript is enabled in chrome.
